python/graphstorm/model/node_glem.py

```diff
@@ -175,12 +181,23 @@ def forward(self, blocks, node_feats, edge_feats, labels, input_nodes, use_gnn=T
         no_pl : bool
             If True, do not calculate pseudo likelihood, use MLE loss only
         """
-        if use_gnn:
-            total_loss = self.forward_gnn(blocks, node_feats, edge_feats, labels, input_nodes,
-                                          no_pl=no_pl)
+        if blocks_u is None or no_pl:
```

Comment:
do you need to test both?

Reply:
Yes. `no_pl` is used to force ignoring unlabeled data for pretraining LM or GNN. This is required even when unlabeled data are provided.